source/documentors/references/doc_style_guide.rst

```diff
@@ -0,0 +1,413 @@
+
+As an example, if a new file ``build_a_course.rst`` is created, then it would need to be appended to an existing table of contents, as shown in the example below. 
+
+Also, you can do a :glob: `*` on an index page, meaning in some cases, documents in a page tree are automatically added to the ToC. In the following example, any file contained in the same folder will automatically be added to the table of contents. It is a quick way to set up a table of contents so that new documents are automatically picked up, but it reduces the control you have on the order of your documents in the TOC:
+
+.. code-block::
+
+    .. toctree::
+       :maxdepth: 1
+       :glob:
+
+       *
+
+Tables
+======
+
+When you include a table, be sure to include a heading row. In addition, consider whether a stub column is appropriate. The heading row and stub column provide useful context for users of screen readers.
+
+Tables should be used to compile complicated data and indicate its relationships. In Open edX documentation, tables look like this:
+
+.. list-table:: Title
+   :widths: 25 25 50
+   :header-rows: 1
+
+   * - Heading row 1, column 1
+     - Heading row 1, column 2
+     - Heading row 1, column 3
+   * - Row 1, column 1
+     -
+     - Row 1, column 3
+   * - Row 2, column 1
+     - Row 2, column 2
+     - Row 2, column 3
+
+When to use a table:
+• When the information is significantly complicated.
+• When the information repeats at least one element more than twice.
+• When the information does not benefit from being compressed.
+
+When not to use a table:
+
+• If that information could be better written in a sentence.
+• The information is important but not sufficiently complicated.
+
+For information on entering tables in RST, see the RST guide: https://docs.openedx.org/en/latest/documentors/references/quick_reference_rst.html
+
+Tags
+====
+
+Tags appear as links (e.g. #Administrator) at the top of the article. If you click on the tag, it will take you to a page listing all files with that tag. This allows users to search for documents with a particular tag. A typical document should have at least two tags:
+- One or more persona tags: The document is tagged with the intended reader persona type. If the document is relevant to more than one audience, then you may use more than one tag. Educator, developer, site operator, community, documentor, translator.
+- One (and only one) diataxis tag: The document is tagged with the diataxis type it belongs to. Concept, reference, quick-start, how-to.
+
+The syntax of tags is a directive, `.. tags::`, followed by the tag names, such as `educators, reference`. It renders as follows:
```

Comment:
```suggestion
The syntax of tags is a directive, `.. tags::`, followed by the tag names, such as `educators, reference`. It renders as follows:
```
```suggestion
The syntax of tags is a directive, ``.. tags::``, followed by the tag names, such as `educators, reference`. It renders as follows:
```

This will fix one of your build errors, can you check out the other reasons why your build is failing?